packages/medusa/src/services/customer.ts

```diff
@@ -23,38 +43,16 @@ class CustomerService extends BaseService {
     eventBusService,
     addressRepository,
   }) {
-    super()
+    // eslint-disable-next-line prefer-rest-params
+    super(arguments[0])
```

Comment:
suggestion: I suggest, that this approach is only used when there is a lot to inject, otherwise it is not really necessary. wdyt?